var/spack/repos/builtin/packages/lcov/package.py

```diff
@@ -18,6 +18,7 @@ class Lcov(MakefilePackage):
 
     license("GPL-2.0-or-later")
 
+    version("master", branch="master", git="https://github.com/linux-test-project/lcov.git")
```

Comment:
```suggestion
    version("master", branch="master")
```

Let's add the `git` property to the package globally instead of on the single version.

```patch
  url = "https://github.com/linux-test-project/lcov/releases/download/v2.0/lcov-2.0.tar.gz"
+ git = "https://github.com/linux-test-project/lcov.git"

  maintainers("KineticTheory")
```

Reply:
Thanks for the feedback.  I've implemented your recommendations.